samples/excel/16-custom-functions/streaming-function.yaml

```diff
@@ -0,0 +1,32 @@
+order: 2
+id: excel-custom-functions-streaming
+name: Streaming function
+description: A streaming function that continuously increments the cell value.
+host: EXCEL
+api_set:
+    CustomFunctionsRuntime: 1.1
+script:
+    content: |
+        /** @CustomFunction 
+        *  @description Increments the cell with a given amount at a specified interval in milliseconds.
+        * @param amount - The amount to add to the cell value on each increment.
+        * @param interval - The time in milliseconds to wait before the next increment on the cell.
+        * @returns An incrementing value.
+        * @streaming
```

Comment:
You can check with Adam, but I'm 99% sure you don't need "@streaming" since you're already marking the handler as a `StreamingInvocation`.

Reply:
But if you do need it, I would move it higher up so that it's more visible/obvious.

Reply:
yep, I removed it.